app.js

```diff
@@ -0,0 +1,12 @@
+
+
+const MyButton = document.querySelector('#randomize');
+const NewImg = function() {
```

Comment:
a way to change the picture is by having an width and height input as a variable that you can use for the picture URL.

```
Get entered width and height values
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");

use this as the URL
let url = ("https://source.unsplash.com/" + width.value + "x" + height.value);
image.setAttribute("src", url);
```